node/src/GlideClient.ts

```diff
@@ -374,6 +375,39 @@ export class GlideClient extends BaseClient {
         return this.createWritePromise(createTime());
     }
 
+    /**
+     * Copies the value stored at the `source` to the `destination` key. If `destinationDB` is specified,
+     * the value will be copied to the databased specified, otherwise the current database will be used.
+     * When `replace` is true, removes the `destination` key first if it already exists, otherwise performs
+     * no action.
+     *
+     * See https://valkey.io/commands/copy/ for more details.
+     *
+     * @param source - The key to the source value.
+     * @param destination - The key where the value should be copied to.
+     * @param destinationDB - (Optional) The alternative logical database index for the destination key.
```

Comment:
We should probably change these docs now that we're using an options parameter. The current doc makes it seem like replace and destinationDB are two separate args to this function. Same for in GlideClusterClient, Transaction

Reply:
I prefer to keep it as `@param desitnationDB` and `@param replace` to help user know how to use these parameters. I have added more example blocks to this command.

Reply:
I also discuss this with Yury. He also agrees to keep these param. I will mark this as resolved.